src/utils/conversion.ts

```diff
@@ -3832,6 +3848,20 @@ export function statTypeToStatOpType(
   }
 }
 
+/**
+ * @hidden
+ */
+export function transferRestrictionTypeToStatOpType(
+  type: TransferRestrictionType,
+  context: Context
+): PolymeshPrimitivesStatisticsStatOpType {
+  if (type === TransferRestrictionType.Count || type === TransferRestrictionType.ClaimCount) {
+    return statisticsOpTypeToStatOpType(StatisticsOpType.Count, context);
+  } else {
```

Comment:
nit -> we don't explicitly need `else` keyword as we are returning from each branch